packages/blitz-auth/src/client/index.tsx

```diff
@@ -259,7 +252,7 @@ export type RedirectAuthenticatedToFn = (
 
 export type BlitzPage<P = {}> = React.ComponentType<P> & {
   getLayout?: (component: JSX.Element) => JSX.Element
-  authenticate?: boolean | {redirectTo?: string | RouteUrlObject; role?: string | Array<string>}
+  authenticate?: boolean | {redirectTo?: string | RouteUrlObject; roles?: string | Array<string>}
```

Comment:
🔨must not change this api
```suggestion
  authenticate?: boolean | {redirectTo?: string | RouteUrlObject; role?: string | Array<string>}
```